Misc/NEWS.d/next/Core and Builtins/2024-03-07-22-38-07.gh-issue-116477.AqwRAV.rst

```diff
@@ -0,0 +1 @@
+Improve performance of the ``range`` function.
```

Comment:
It is not clear what operations were sped up. If I understand it correctly, it is something like:
```suggestion
Speed up the :class:`range` object constructor and the :class:`!range`
iterator constructor in the case of a single argument.
```